doc/ref_model/chapters/chapter04.md

```diff
@@ -532,18 +532,13 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
 | `e.nfvi.mon.cap.001`<br />(Monitoring of L2-7 data) | No | Yes | No | Exposed monitoring capabilities as per [**Table 4-3**](#Table4-3) |
 | `i.nfvi.sla.cap.001`<br />(CPU overbooking) | 1:4 | 1:1 | 1:1 | Internal SLA capabilities as per [**Table 4-6**.](#Table4-6) |
 | `i.nfvi.sla.cap.002`<br />(vNIC QoS) | No | Yes | Yes | |
-| `i.nfvi.per.cap.001`<br />(Huge page support) | No | Yes | No | Internal performance capabilities as per [**Table 4-7**](#Table4-7) |
+| `i.nfvi.per.cap.001`<br />(Huge page support) | No | Yes | Yes | Internal performance capabilities as per [**Table 4-7**](#Table4-7) |
 | `i.nfvi.mon.cap.001`<br />(Host CPU usage) | Yes | Yes | Yes | Internal monitoring capabilities as per [**Table 4-8**](#Table4-8) |
 | `i.nfvi.mon.cap.002`<br />(Virtual compute CPU usage) | Yes | Yes | Yes | |
 | `i.nfvi.mon.cap.003`<br />(Host CPU utilization) | Yes | Yes | Yes | |
 | `i.nfvi.mon.cap.004`<br />(Virtual compute CPU utilization) | Yes | Yes | Yes | |
-| `i.nfvi.mon.cap.005`<br />(External storage IOPS) | Yes | No | Yes | |
-| `i.nfvi.mon.cap.006`<br />(External storage throughput) | Yes | No | Yes | |
-| `i.nfvi.mon.cap.007`<br />(External storage capacity) | Yes | No | Yes | |
-| `i.nfvi.sec.cap.001`<br />(VNFC/VNFC memory isolation) | Yes | Yes | Yes | Internal security capabilities as per [**Table 7-1**](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter07.md#Table7-1) |
-| `i.nfvi.sec.cap.002`<br />(VNFC -> Host) | No | No | No | |
-| `i.nfvi.sec.cap.003`<br />(Host -> VNFC) | Yes | Yes | Yes | |
```

Comment:
Has there been any follow-up WRT the points @kedmison raised on Issue #385? I think they have to be addressed before these lines can be deleted. I'm good w/ the rest of the edit.